src/EnergyPlus/DXCoils.cc

```diff
@@ -12352,7 +12354,11 @@ namespace DXCoils {
                         WasteHeatHS = CurveValue(DXCoil(DXCoilNum).MSWasteHeat(SpeedNumHS), OutdoorDryBulb, InletAirDryBulbTemp) *
                                       DXCoil(DXCoilNum).MSWasteHeatFrac(SpeedNumHS);
                     }
-                    MSHPWasteHeat = (SpeedRatio * WasteHeatHS + (1.0 - SpeedRatio) * WasteHeatLS) * DXCoil(DXCoilNum).ElecCoolingPower;
+                    DXCoil(DXCoilNum).MSFuelWasteHeat =
+                        (SpeedRatio * WasteHeatHS + (1.0 - SpeedRatio) * WasteHeatLS) * DXCoil(DXCoilNum).ElecCoolingPower;
```

Comment:
It's confusing here (and below) to see `MSFuelWasteHeat` calculated using `ElecCoolingPower` (or `ElecHeatingPower` below).

Reply:
@mjwitte Originally the DataHVACGlobals::MSHPWasteHeat is calculated as:
```
MSHPWasteHeat = (SpeedRatio * WasteHeatHS + (1.0 - SpeedRatio) * WasteHeatLS) * DXCoil(DXCoilNum).ElecCoolingPower;                  
```
I was trying to use this and assign it as a DXCoil variable.

I now realize this global waste heat is calculated with each DXCoilNum, overwriting one another instead of aggregating, which is weird to me.